src/HexagonOffload.cpp

```diff
@@ -60,7 +60,7 @@ Stmt replace_params(Stmt s, const std::map<std::string, Parameter> &replacements
 class InjectHexagonRpc : public IRMutator {
     std::map<std::string, Expr> state_vars;
 
-    Module device_code;
+    Module &device_code;
```

Comment:
Why is this a reference? Isn't a Module already a shared pointer to something...